### ISSUE TITLE
Video description not included in search

### ISSUE THREAD

**Issue by prepperz** [PHPfox-Official/phpfox-v4-issues#2510]
## What happened?
Video description not included in search

## Steps to reproduce:
1. search any word in video description

## What is expected?
...no results found

## Browsers and Devices tested
Chrome on Windown 10

## Server information
24 Cores, CentOS 7.6.1810, php7.0.27, apache 2.4.6, MySQL: 5.5.60-MariaDB

...

## phpFox version
phpFox 4.7.1 (Upgraded), Material theme

...

## Screenshots

**Comment by harrison05** [comment]
We only support search Videos by title

Regards.

**Comment by prepperz** [comment]
Is there a list of what the search parameters include?

**Comment by spirogg** [comment]
@harrison05 we need tags users can add in video and use tye tags parameter also to search with, 
 like YouTube and other video upload scripts have.

**Comment by prepperz** [comment]
Content should definitely be included in search.

**Comment by harrison05** [comment]
> @harrison05 we need tags users can add in video and use tye tags parameter also to search with,
> like YouTube and other video upload scripts have.

@spirogg Currently, a user can click on videos hashtags in content to search. Do you want Video App to allow user type hashtag and search in Search Box like YouTube?
![screen shot 2019-01-02 at 10 07 43](https://user-images.githubusercontent.com/41947198/50579281-3df66b00-0e76-11e9-97d0-97aa54d2955f.png)
@prepperz We will review your request.

Regards.

**Comment by spirogg** [comment]
> @spirogg Currently, a user can click on videos hashtags in content to search. Do you want Video App to allow user type hashtag and search in Search Box like YouTube?

@harrison05 i would like when you add a video we can add tags and tags to be included to search with

see photo when you upload video in youtube you can add Title, Description and beneath that we can add tags.

also when users or admin use youtube url to upload (import a video from youtube) we only import title and description, but we can also import tags. and use tags to search with. 

When uploading a video - I would also like to be able to add:
Title ( we have this option)
Description (we have this option)
Tags ( we don;t have this option)    NOT hashtags just Tags

![tags](https://user-images.githubusercontent.com/17736688/50581969-37d79e80-0e24-11e9-8c76-cd947786386c.PNG)

**Comment by harrison05** [comment]
@spirogg Yes, I got it, you want to support Tags for Video like Topics of Blogs/Forums 👍
